core/spec/models/spree/order_spec.rb

```diff
@@ -537,7 +537,7 @@ def compute(computable)
     let(:methods) { [ok_method, no_method] }
 
     it "does not include a payment method that is not suitable for this order" do
-      allow(Spree::PaymentMethod).to receive(:available_on_front_end).and_return(methods)
+      allow(Spree::PaymentMethod).to receive(:available_on_front_end, :within_transaction_limits).and_return(methods)
```

Comment:
@krtschmr I've investigated this failing spec, and it seems that the `ok_method` double received the `within_transactional_limits?` call, which the double was not expecting. if you add the following in line 540: `allow(ok_method).to receive(:within_transaction_limits?).with(order).and_return(true)` the test will pass. :)
The whole spec should look like this:
```
it "does not include a payment method that is not suitable for this order" do
      allow(ok_method).to receive(:within_transaction_limits?).with(order).and_return(true)
      allow(Spree::PaymentMethod).to receive(:available_on_front_end).and_return(methods)

      expect(order.available_payment_methods).to match_array [ok_method]
    end
```